### DISCUSSION TITLE
小程序授权调用时提示codeToSession方法不存在

### DISCUSSION THREAD

**Discussion by xgbnl** [w7corp/easywechat#2538] [Q&A]
####当前版本:"w7corp/easywechat": "^6.7"
![image](https://user-images.githubusercontent.com/52394568/178616575-fae10382-cf8d-4e50-8d07-e61f98a79001.png)

![image](https://user-images.githubusercontent.com/52394568/178616673-3d4dd07b-d7b5-4be7-b129-932fdf5b31ec.png)

**Comment by xgbnl** [comment]
看源码也是没有Util类

**Comment by xgbnl** [comment]
已解决

**Reply by dux-web** [reply]
怎么解决的？
